laws/src/main/scala/cats/laws/InvariandAddMonoidalLaws.scala

```diff
@@ -0,0 +1,30 @@
+package cats
+package laws
+
+import cats.data.INothing
+
+/**
+ * Laws that must be obeyed by any `cats.InvariantChoosable`.
+ */
+trait InvariantChoosableLaws[F[_]] extends InvariantChoiceLaws[F] {
+  implicit override def I: InvariantChoosable[F]
+  import cats.syntax.invariant._
+
+  def sumEmptyLeftIdentity[A, B](fa: F[A]): IsEq[F[A]] =
```

Comment:
Could this be
```suggestion
  def choiceEmptyLeftIdentity[A, B](fa: F[A]): IsEq[F[A]] =
```
and the below `choiceEmptyRightIdentity`